R/dstrace.R

```diff
@@ -1,17 +1,20 @@
 
 #' dstrace
 #'
-#' @param object_name Name of object to trace
-#' @param all_data Data structure produced by a call to \code{\link{driver}}
+#' @param object_name Name of object to trace (can be either a data object or a code chunk)
+#' @param gcam_data_map A tibble of metadata information; normally a built-in package dataset
 #' @param previous_tracelist Information about previous objects printed
 #' @param recurse Recurse to print information about precursor objects?
-#' @return Boolean (object_name found, or not)
+#' @return A tibble with the trace information (object name and trace number)
```

Comment:
I wonder if there's any kind of option to return a plot as well?  
If I'm trying to figure out a problem, especially with something that results in as many objects as `dstrace("demand_input.xml")`, being able to effortlessly plot a directed graph might be really helpful. Maybe default to not doing it, but have an option like `dstrace("demand_input.xml", plot_directed_graph = TRUE)`

Reply:
That would definitely be desirable, agreed. Particularly if the graph were labeled with the corresponding numbers & names that appear in the `dstrace` output!

Opening new issue for this.